db/types.go

```diff
@@ -21,6 +21,8 @@ const (
 	TxTypeSettlement               string = "SETTLEMENT"
 	TxTypeHyperlaneMessageDelivery string = "HYPERLANE_MESSAGE_DELIVERY"
 	TxTypeInitiateTimeout          string = "INITIATE_TIMEOUT"
+	TxTypeERC20Approval            string = "ERC20_APPROVAL"
```

Comment:
may have to revert the commit that copied the work from the tx profit tracking pr, not 100% sure tho but want to call it out